sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIService.scala

```diff
@@ -73,15 +73,15 @@ private[hive] class SparkSQLCLIService(hiveContext: HiveContext)
 }
 
 private[thriftserver] trait ReflectedCompositeService { this: AbstractService =>
-  def initCompositeService(hiveConf: HiveConf) {
+  def initCompositeService(hiveConf: HiveConf, level: Int = 2) {
```

Comment:
I'd suggest moving the whole `SparkSQLSessionManager` class into the shim layer for now. I'm planning to add a reflection based utility that can call 0.12.0 / 0.13.1 APIs adaptively and remove most contents out of the shim layer later.